src/components/HTMLEngineProvider/HTMLRenderers/MentionReportRenderer.tsx

```diff
@@ -63,7 +62,12 @@ function MentionReportRenderer({style, tnode, TDefaultRenderer, reports, ...defa
 
     const currentReportID = useCurrentReportID();
     const currentReport = getReport(currentReportID?.currentReportID);
-    const isGroupPolicyReport = useMemo(() => (currentReport && !isEmptyObject(currentReport) ? ReportUtils.isGroupPolicy(currentReport) : false), [currentReport]);
+
+    // When we invite someone to a room they don't have the policy object, but we still want them to be able to see and click on report mentions, so we only check if the policyID in the report is from a workspace
+    const isGroupPolicyReport = useMemo(
+        () => (currentReport && !isEmptyObject(currentReport) ? !!currentReport.policyID && currentReport.policyID !== CONST.POLICY.ID_FAKE : false),
```

Comment:
I find the ternary here a bit hard to follow. A ternary seems like a weird choice for this type of condition. Is this the same as saying? 
```
(currentReport && !isEmptyObject(currentReport) && !!currentReport.policyID && currentReport.policyID !== CONST.POLICY.ID_FAKE : false),
```
(currentReport && !isEmptyObject(currentReport) && !!currentReport.policyID && currentReport.policyID !== CONST.POLICY.ID_FAKE
```

Reply:
It is, thanks